test_files/lobster/vasp_lobster_output/FW.json

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
Is this file even necessary. Custodian stands separate from Fireworks.

Reply:
will remove this as well.